### ISSUE TITLE
Question: how close will this project be to datapusher?

### ISSUE THREAD

**Issue by categulario** [dathere/datapusher-plus#7]
Are they meant to diverge or is it expected that stuff that is merged to datapusher will keep being merged here?

I think such an issue is important to consider because some great improvements can be made to `datapusher-plus` if it properly diverges from `datapusher`. Decisions made there no longer would have to affect here, for instance a proper package install can be thought of, or things like a container image.

How likely is it that datapusher will be replaced with datapusher-plus if it outperforms it? What is the relationship between the maintainers of this project and those of datapusher (and ckan in general)?

I know, a lot of questions, but it would give me some context and an idea of how I can contribute

**Comment by jqnatividad** [comment]
@categulario - Rest assured, **Datapusher+ will _always_ be a drop-in replacement for Datapusher.**

It will continue to leverage CKAN Service Provider, and track relevant upstream changes from Datapusher (e.g. [I adapted your init_db PR upstream for instance](https://github.com/dathere/datapusher-plus/pull/3) and just merged your [JOB_CONFIG pr](#6 ). 😉 ).

**We will always look to contribute upstream by default.**  That has been our track record while we Ontodia, after we were acquired by OpenGov (where we sponsored several contributions, including ckanext-xloader), and now that we're independent again with datHere - as we firmly believe in open source principles for all the reasons we're all familiar with, but in particular for Datapusher+ - the open-source truism of "**_permissionless innovation_**."

That's one reason why we have a detached fork - our aim is to greatly increase the iteration velocity (witness qsv - another detached fork we maintain and its [frequent releases](https://github.com/jqnatividad/qsv/releases)) - in terms of having our own issue tracker, pull requests, discussions, etc. 

We've been [pawing at several Datapusher itches since 2015](https://github.com/ckan/ideas/issues/150), and a recent project has finally given me the use cases and most importantly - the support to scratch them in earnest. 😄 

Our first startup - Ontodia was honored to be appointed the first CKAN professional services provider in North America in 2013, where we partnered with the CKAN team responding to several procurements in the US. When Ontodia was acquired by OpenGov in 2016, I was honored to accept the nomination to the CKAN Steering Group and we did our best to contribute and sponsor several CKAN features and extensions while we were there. 

Now that we're a scrappy startup again at datHere, we intend to apply all the hard lessons learned in our 12-year open data/open source journey, and double-down on open source, and do our best to be active contributors and help foster a vibrant CKAN ecosystem - as we see a dire need to create a [standards-based, open source Data Infrastructure](https://dathere.com/2021/08/what-is-data-infrastructure/).

As for Datapusher+ replacing Datapusher - that's not our decision alone, but should the core team and the CKAN custodians want to do so - there's nothing preventing it, and we would be more than happy to co-maintain it as a core CKAN component.

So yes! Please keep `em PRs, suggestions, ideas, feedback coming!!!

**Comment by jqnatividad** [comment]
BTW @categulario , I think this thread is best moved to Discussions, so it doesn't get lost in the Issue tracker, which is primarily meant for bug reports and feature requests.  Is it OK if I move it there?

**Comment by categulario** [comment]
Yeah, let's move it to discussions. I agree that's a better place
